README.md

```diff
@@ -24,17 +24,7 @@ docker run \
 	--rm \
 	--link='smr-mysql' \
 	--volume="$(pwd)/db/patches:/flyway/sql:ro" \
-	shouldbee/flyway \
-	-url='jdbc:mysql://smr-mysql/smr_live' \
-	-user='smr' \
-	-password='smr' \
-	init
```

Comment:
Doesn't `init` still need to be run the first time?

Reply:
Nope! Not since version 4.0.